### DISCUSSION TITLE
Can it be create a manual Bundle with the webpack?

### DISCUSSION THREAD

**Discussion by logicchemie** [webpack/webpack#16342] [Q&A]
I have scripts and styles in variables:
```
let js_script_1 = `javascript code ...`;
let js_script_2 = `javascript code ...`;
let css_style_1 = `css ...`;
```
can it be create a manual bundle.js file with the webpack?

**Comment by alexander-akait** [comment]
You can create a custom plugin for this, but why don't use file and load them using `import(...)`? You mark `import(...)` as `eager` so it will not create new chunk

**Reply by logicchemie** [reply]
My project is on the server side.
And I fetching styles and scripts from database.
Can I use webpack as a code optimization tool?

**Reply by alexander-akait** [reply]
Not sure you should use webpack here, storing assets (js/css/images/etc) in a database is not a good idea

**Reply by logicchemie** [reply]
I agree with you.
i want to save bundle.js as file system,
and when the page is loaded, the request is not sent to the database.
(just for js/css)
Can I use webpack as a code optimization tool?

**Reply by alexander-akait** [reply]
You can write a script to fetch all files from database and then use webpack to bundle them

**Reply by samagragupta** [reply]
Will webpack not do bundling during build time only
